include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -32,7 +32,7 @@ namespace seqan3
 {
 
 /*!\brief A space-optimised version of std::vector that compresses multiple letters into a single byte.
- * \tparam alphabet_type The value type of the container, must satisfy seqan3::alphabet and not be `&`.
+ * \tparam alphabet_type The value type of the container, must satisfy seqan3::writable_semialphabet and std::regular.
```

Comment:
not be `&` rather translates to `std::semi_regular` no?

Reply:
alphabet and and semiregular is automatically regular.